### PR TITLE
(RE-5578) Sign and ship swix packages

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

File: tasks/ship.rake [hunk 1]
```diff
@@ -147,6 +147,17 @@ namespace :pl do
     end
   end
 
+   desc "ship tarball and signature to #{Pkg::Config.tar_host}"
+   task :ship_swix => 'pl:fetch' do
+    if Dir['pkg/eos/**/*.swix'].empty?
```
Comment (haus) [inline tasks/ship.rake]: The tabbing is off here.

File: tasks/ship.rake [hunk 2]
```diff
@@ -147,6 +147,17 @@ namespace :pl do
     end
   end
 
+   desc "ship tarball and signature to #{Pkg::Config.tar_host}"
```
Comment (haus) [inline tasks/ship.rake]: copypasta

File: tasks/ship.rake [hunk 3]
```diff
@@ -147,6 +147,17 @@ namespace :pl do
...
+   task :ship_swix => 'pl:fetch' do
+    if Dir['pkg/eos/**/*.swix'].empty?
+      STDOUT.puts "There aren't any swix packages in pkg/eos. Maybe something went wrong?"
+    else
+     Pkg::Util::Execution.retry_on_fail(:times => 3) do
+       Pkg::Util::Net.rsync_to("pkg/#{Pkg::Config.project}-#{Pkg::Config.version}.tar.gz*", Pkg::Config.tar_host, Pkg::Config.tarball_path)
```
Comment (haus) [inline tasks/ship.rake]: copypasta

File: tasks/ship.rake [hunk 4]
```diff
@@ -155,6 +166,7 @@ namespace :pl do
       Rake::Task["pl:ship_rpms"].invoke if Pkg::Config.final_mocks || Pkg::Config.vanagon_project
       Rake::Task["pl:ship_debs"].invoke if Pkg::Config.cows || Pkg::Config.vanagon_project
       Rake::Task["pl:ship_dmg"].execute if Pkg::Config.build_dmg || Pkg::Config.vanagon_project
+      Rake::Task["pl:ship_swix"].execute if Pkg::Config.build_swix || Pkg::Config.vanagon_project
```
Comment (haus) [inline tasks/ship.rake]: I don't think we need a build_swix flag here (or the param in general), as all swix packages are vanagon projects.

File: lib/packaging/config/params.rb [hunk 5]
```diff
@@ -16,6 +16,7 @@ module Pkg::Params
                   :build_ips,
                   :build_msi,
                   :build_pe,
+                  :build_swix,
```
Comment (haus) [inline lib/packaging/config/params.rb]: As I mentioned below, I don't think we need this flag as the packaging repo doesn't have the capacity to build swix packages.

**Comment by melissa** [comment]
Requires https://github.com/puppetlabs/build-data/pull/29

**Comment by melissa** [comment]
Signing and shipping both work

**Comment by haus** [comment]
@melissa do you mind adding a bit more content to the commit messages?

**Comment by melissa** [comment]
@haus better?

**Comment by haus** [comment]
@melissa much

**Comment by haus** [comment]
:+1:

**Pull request merged**